### PR TITLE
Kraken: Move set_valid_jp_and_jpp outside of the RAPTOR loop

### PR DESCRIPTION
**Pull request by nonifier**

### CODE REVIEW

**Comment by nonifier** [comment]
From my quick profiling using perf, I go from
   0,40%  kraken [.] _ZN7navitia7routing6RAPTOR20set_valid_jp_and_jpp...
to
   0,02%  kraken [.] _ZN7navitia7routing6RAPTOR20set_valid_jp_and_jpp...

(time spent in the function).

**Review by TeXitoi — APPROVED** [review]

**Review by benoit-bst — APPROVED** [review]

**Comment by nonifier** [comment]
jenkins, test this please !

File: source/routing/raptor.cpp [hunk 1]
```diff
@@ -535,23 +540,17 @@ RAPTOR::isochrone(const map_stop_point_duration& departures,
...
-    const DateTime bound = limit_bound(clockwise, departure_datetime, b);
+                  const nt::RTLevel rt_level)
+{
     set_valid_jp_and_jpp(DateTimeUtils::date(departure_datetime),
                          accessibilite_params,
                          forbidden,
                          allowed,
                          rt_level);
-    assert(data.dataRaptor->cached_next_st_manager);
-    next_st = data.dataRaptor->cached_next_st_manager->load(
```
Comment (xlqian) [inline source/routing/raptor.cpp:546]: so the next stoptimes cache is removed? am i missing something?

Comment (nonifier) [reply source/routing/raptor.cpp:546]: You're missing that the function now calls `first_raptor_loop()` that do the caching ;-)

**Review by xlqian — APPROVED** [review]

**Pull request merged**